backend/src/test/java/controller/RoleControllerTest.java

```diff
@@ -0,0 +1,195 @@
+package controller;
+
+
+import com.beaplab.BeaplabEngine.controller.RoleController;
+import com.beaplab.BeaplabEngine.metadata.RoleDto;
+import com.beaplab.BeaplabEngine.service.RoleService;
+import com.rollbar.notifier.Rollbar;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+import static utils.MockFactory.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RoleControllerTest {
+
+    @InjectMocks
+    private RoleController roleController;
+
+    @Mock
+    private Rollbar rollbar;
+
+    @Mock
+    private RoleService roleService;
+
+
+    /**
+     * T5.77
```

Comment:
Please add pre conditions and post conditions to the tests

Reply:
fixed